Week_7.java

```diff
@@ -0,0 +1,166 @@
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.Map;
+
+/*
+
+Microsoft Interview Coaching Pilot 2018 (MICP 2018)
+Author: Aishna Gupta
+Language: JAVA
+
+************WEEK 6************
+
+Problem description: Design a vending machine.
+Clarify: No more restrictions, just design a general vending machine we are familiar with.
+*/
+
+public class Week_7 {
+	
+	
+	public class Item{
+		private String Item_Id;
+		private String Item_Name;
+		private int Item_Quantity;
+		private int Item_Price;
+		
+		Item(String id, String name, int quantity,int price){
+			Item_Id = id;
+			Item_Name = name;
+			Item_Quantity = quantity;
+			Item_Price = price;
```

Comment:
There should probably be some restrictions on these parameters, non null or empty Id & Name.  quantity and price > 0

Reply:
Same advice when you're setting the values below.  If someone tries to create and invalid Item, or set the value to something invalid, you should throw an exception